cpp/include/games/blokus/Game.hpp

```diff
@@ -56,7 +57,7 @@ class Game {
   using Types = core::GameTypes<Constants, State, GameResults, SymmetryGroup>;
   using Symmetries = core::TrivialSymmetries;
 
-  struct Rules {
+  struct Rules : public game_base::RulesBase<Types, State> {
```

Comment:
`Types` already is passed `State` as a template param, so it's redundant to pass it again to `RulesBase`.